pom.xml

```diff
@@ -163,8 +163,15 @@
     <build>
         <resources>
             <resource>
-                <filtering>false</filtering>
                 <directory>rules</directory>
+                <filtering>false</filtering>
+            </resource>
+            <resource>
+                <directory>rules-reviewed/</directory>
```

Comment:
See the discussion above - it simply makes that directory treated equally by the IDE as rules/ .